<?xml version="1.0"?>
<launch>
        <arg name="waypoints_not_trajectories" default="false"/>

        <group if="$(arg waypoints_not_trajectories)">
            <node pkg="forgetful_drones" type="forgetful_simulator" name="forgetful_simulator" output="screen">
                <rosparam file="$(find forgetful_drones)/parameters/forgetful_simulator.yaml" />
                <param name="drone_model_description" value=""/>

                <param name="racetrack_data_generation_on" value="true"/>
                <param name="debug_on" value="false"/>
            </node>
        </group>

        <group unless="$(arg waypoints_not_trajectories)">
            <node pkg="forgetful_drones" type="forgetful_drone" name="forgetful_drone" output="screen">
                <rosparam file="$(find forgetful_drones)/parameters/forgetful_drone.yaml" />

                <param name="racetrack_data_generation_on" value="true"/>
                <param name="debug_on" value="false"/>
            </node>
        </group>


</launch>

